addons/crm_iap_mine/static/src/views/generate_leads_views.js

```diff
@@ -19,11 +19,11 @@ registry.category("views").add("crm_iap_lead_mining_request_tree", {
     Controller: LeadMiningRequestListController,
     buttonTemplate: "LeadMiningRequestListView.buttons",
 });
-
+// why patch it and not replace it in the registry ?
```

Comment:
replacing in the registry doesn't allow composition, ie 2 patches will be mutually exclusive. A patch is more robust.